docs/telegram-usage.md

```diff
@@ -28,6 +28,8 @@ official commands. You can ask at any moment for help with `/help`.
 | `/performance` | | Show performance of each finished trade grouped by pair
 | `/balance` | | Show account balance per currency
 | `/daily <n>` | 7 | Shows profit or loss per day, over the last n days
+| `/whitelist` | | Show the current whitelist
+| `/blacklist [pair]` | | Show the current blacklist, or adds a pair to the blacklist.
```

Comment:
does it make sense to make it consistent ? adding `/whitelist [pair]` maybe ? cause if I add a pair on BL, the only way to remove is to /reload_conf

Reply:
adding a pair to whitelist is not the opposite of removing it from blacklist.

Maybe better would be `/blacklist remove <pair>` ... but I'm not sure on the usage, it's to quickly remove a pair "on the go", and then add it permanently "when at home" ... or so